lib/chainweb/p2p/block_payload.ex

```diff
@@ -26,4 +30,27 @@ defmodule Kadena.Chainweb.P2P.BlockPayload do
     |> Request.perform()
     |> Request.results(as: BlockPayloadResponse)
   end
+
+  @spec retrieve_batch(payload_hashes :: payload_hashes(), network_opts :: network_opts()) ::
+          retrieve_batch_response()
+  def retrieve_batch(payload_hashes \\ [], network_opts \\ []) do
```

Comment:
Why set a default value for `payload_hashes`?

Reply:
Because a request like this:
payload_hashes : [ ]
is a valid request that returns as a response :[ ]
![image](https://user-images.githubusercontent.com/80716239/214094475-678660be-be15-45ef-ab82-b012494b6f05.png)